lib/apivore/swagger_checker.rb

```diff
@@ -42,10 +44,12 @@ def base_path
       @swagger.base_path
     end
 
-    attr_reader :swagger_path, :mappings, :swagger
+    attr_reader :swagger_path, :untested_mappings, :swagger
 
     private
 
+    attr_reader :mappings
```

Comment:
Gottcha, :untested_mappings now provides the same data :mappings used to publicly, no need to add something else :+1: